src/open_inwoner/templates/pages/search.html

```diff
@@ -11,30 +11,29 @@ <h1 class="h1">{% trans "Zoeken naar " %} "{% firstof search_form.query.value ""
     </div>
 
     {#  facets and results section #}
-    {% if paginator.count %}
-        <div class="grid">
-            <aside class="grid__sidebar grid__filters" aria-label="{% trans "Zoekfilters" %}">
-            {% if search_filter_categories or search_filter_tags or search_filter_organizations %}
-                <h2 class="h2">{% trans "Zoekfilters" %}</h2>
-                    {% if search_filter_categories %}
-                        {% filters field=search_form.categories form_id="search-form" %}
-                    {% endif %}
-                    {% if search_filter_tags %}
-                         {% filters field=search_form.tags form_id="search-form" %}
-                    {% endif %}
-                    {% if search_filter_organizations %}
-                        {% filters field=search_form.organizations form_id="search-form" %}
-                    {% endif %}
-            {%  endif %}
-            {#  end search filters #}
-            </aside>
+    <div class="grid">
```

Comment:
Ah nice - does this mean it's no longer dependent on the pagination?

Reply:
Not 100% sure what you mean but with this change the facet checkboxes always show even when zero results.

The reason the "do we have results" check used the paginator is because the paginator was conveniently available in the context.